app/controllers/export_controller.rb

```diff
@@ -169,7 +169,7 @@ def full_history_patient
       }
     }
 
-    send_data write_export_data_to_files(config, patients, nil, 1)[0][:content]
+    send_data Base64.encode64(write_export_data_to_files(config, patients, 1)[0][:content].read)
```

Comment:
Why is this now being encoded? Were you seeing any issues with this?

Reply:
The content was always encoded. I removed the encoding from the upload to object storage but it is still required for single patient exports: https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/components/subject/DownloadMonitoree.js#L28

Reply:
If you want we can rewrite this to not encode since it adds overhead